### DISCUSSION TITLE
Domestic sites SNI

### DISCUSSION THREAD

**Discussion by bart3nder** [XTLS/Xray-core#2475] [Q&A] [answered]
Hi! Quick question, Is there any risk and security concerns of using domestic sites SNI for REALITY? Or we should aim for foreign websites? I was thinking If REALITY gets the ssl cert from `dest`, Can they spot our server's IP on the `dest` site logs?

**Comment by chika0801** [comment, marked as answer]
In China, for Chinese netizens, the suggestion is that REALITY's DEST fills in the URL, and the website's server should not be in mainland China. At the same time the chosen website should not be blocked by China's GFW firewall. These sites should not have pornography, gambling, scamming or any other content that is not allowed by the government. At the same time in mainland China, can not use the proxy server, direct access to the site.

This is our recommended principle. According to this reasoning, I think if you are in Iran, this principle please refer to it.

**Reply by bart3nder** [reply]
Thanks for the answer, Great as always.
